### ISSUE TITLE
Nullpointer when trying to parse manifest file

### ISSUE THREAD

**Issue by muhammedashraf9244** [wt-io-it/odoo-pycharm-plugin#115]
Plugin Version: 0.6.9
IntelliJ Version: PyCharm 2022.3.1
IntelliJ Build: PC-223.8214.51

<details><summary>java.lang.NullPointerException: Cannot invoke "com.intellij.psi.PsiFile.getProject()" because "manifestFile" is null</summary>

```
java.lang.NullPointerException: Cannot invoke "com.intellij.psi.PsiFile.getProject()" because "manifestFile" is null
	at at.wtioit.intellij.plugins.odoo.modules.impl.OdooManifestParser.parse(OdooManifestParser.java:26)
	at at.wtioit.intellij.plugins.odoo.modules.index.OdooDeserializedModuleImpl.getDependencies(OdooDeserializedModuleImpl.java:83)
	at at.wtioit.intellij.plugins.odoo.modules.impl.AbstractOdooModuleImpl.dependsOn(AbstractOdooModuleImpl.java:62)
	at at.wtioit.intellij.plugins.odoo.records.impl.OdooRecordServiceImpl.lambda$getRecord$1(OdooRecordServiceImpl.java:61)
	at at.wtioit.intellij.plugins.odoo.WithinProject.call(WithinProject.java:15)
	at at.wtioit.intellij.plugins.odoo.records.impl.OdooRecordServiceImpl.getRecord(OdooRecordServiceImpl.java:53)
	at at.wtioit.intellij.plugins.odoo.AbstractOdooCompletionContributor.suggestRecordXmlId(AbstractOdooCompletionContributor.java:117)
	at at.wtioit.intellij.plugins.odoo.OdooCompletionContributor.fillCompletionVariants(OdooCompletionContributor.java:52)
	at com.intellij.codeInsight.completion.CompletionService.getVariantsFromContributor(CompletionService.java:81)
	...
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:246)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	...
	at java.base/java.lang.Thread.run(Thread.java:833)
```
</details>

**Comment by bigbear3001** [comment]
this is fixed with 0.6.11 (pending jetbrains approval) if you are eager to test it you can use the jar file from: https://github.com/wt-io-it/odoo-pycharm-plugin/tree/v0.6.11

**Comment by ap-wtioit** [comment]
0.6.11 is now available in the marketplace
